sdk/servicebus/service-bus/tsconfig.json

```diff
@@ -2,21 +2,15 @@
   "extends": "../../../tsconfig",
   "compilerOptions": {
     "target": "ES2020",
-    "module": "ES2020",
-    "declarationDir": "./types",
-    "outDir": "./dist-esm",
+    "module": "NodeNext",
     "lib": ["dom", "ES2018.AsyncIterable"],
```

Comment:
should we remove those now?

Reply:
we probably could

Reply:
"dom" is needed for some browser types in code to get runtime info. Probably should use core-util for that but not in this PR.

Reply:
How about "dom"? I don't think we need dom here and it should be sufficient to have it in the browser one only, no?

Reply:
yes we can do that too

Reply:
Use stubs now.